### PR TITLE
doc: document used permissions

### LINKED ISSUE #26: Document required permissions
We need to document all required permissions:
https://github.com/rugk/unicodify/blob/main/assets/texts/en/permissions.md

That is linked in the privacy policy.

For a template, look like it is done here:
https://github.com/rugk/offline-qr-code/blob/master/assets/texts/en/permissions.md

### PR DESCRIPTION
**Pull request by rugk**
Fixes #26

### CODE REVIEW

**Review by tdulcet — CHANGES_REQUESTED** [review]
Thanks for doing this. Thunderbird also has an additional permission: https://github.com/rugk/unicodify/blob/ea3adae732fde521e940487486fa6cd7cfa9beb4/scripts/manifests/thunderbirdmanifest.json#L41 See [here](https://thunderbird-webextensions.readthedocs.io/en/latest/compose.html#permissions) for a description.

File: assets/texts/en/permissions.md [hunk 1]
```diff
@@ -4,20 +4,22 @@ For a general explanation of add-on permission see [this support article](https:
 
 ## Installation permissions
 
-Currently, no permission is requested at the installation or when updating.
+The following permissions are requested at the installation or when updating:
 
-## Feature-specific (optional) permissions
+| Internal Id  | Permission                        | Explanation                      |
+|:-------------|:----------------------------------|:---------------------------------|
+| `<all_urls>` | Access your data for all websites | Needed for input auto-correction |
```
[the remark this answers is not in the document]
Comment (rugk) [reply assets/texts/en/permissions.md:11]: Oh thanks, okay, it’s apparently this:

https://support.mozilla.org/de/kb/permission-request-messages-thunderbird-extensions#w_read-and-modify-your-email-messages-as-you-compose-and-send-them

(I want to use the same message as displayed in the permission prompt, that’s why I’m looking that up in the help there.)

**Comment by rugk** [comment]
Oh thanks, okay, it’s apparently this:

https://support.mozilla.org/de/kb/permission-request-messages-thunderbird-extensions#w_read-and-modify-your-email-messages-as-you-compose-and-send-them

(I want to use the same message as displayed in the permission prompt, that’s why I’m looking that up in the help there.)

File: assets/texts/en/permissions.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rugk) [reply assets/texts/en/permissions.md:11]: Oh yeah, maybe, at least when the auto-preview is enabled. I.e. when you only want to react on a click AFAIK `activeTab` or so was enough IIRC.

Anyway, I’ll add it…

**Review by tdulcet — APPROVED** [review]

**Pull request merged**